.github/workflows/pr-push.yml

```diff
@@ -19,5 +19,5 @@ jobs:
           W3C_ECHIDNA_TOKEN: ${{ secrets.W3C_TR_TOKEN }}
           W3C_WG_DECISION_URL: https://lists.w3.org/Archives/Public/public-device-apis/2021May/0008.html
           W3C_BUILD_OVERRIDE: |
-            specStatus: WD
+            specStatus: CRD
```

Comment:
Should we add https://respec.org/docs/#crEnd to respecConfig in https://github.com/w3c/compute-pressure/blob/main/index.html#L27 or can we use `W3C_BUILD_OVERRIDE` also for `crEnd`?

Reply:
I suppose CRD does not have crEnd,, no?

Reply:
That matches my understanding.

I'm just wondering what needs to be done to fix this error message: https://github.com/w3c/compute-pressure/actions/runs/14637232487/job/41071059579?pr=314#step:3:921

Reply:
I haven't realized, but crEnd seems mandatory per pubrules
https://www.w3.org/pubrules/doc/rules/?profile=CRD#reviewEndDate

so, let me update this PR with adding crEnd as the same as 1st CRS publication